<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-5PS7DEO7" failures="1" tests="7" name="ictInternshipPortal.LoginTC" time="68.649" errors="0" timestamp="2024-07-21T18:09:14 IST" skipped="0">
  <testcase classname="ictInternshipPortal.LoginTC" name="TC_IP_1_1_1" time="0.062"/>
  <testcase classname="ictInternshipPortal.LoginTC" name="TC_IP_1_2_1" time="1.866"/>
  <testcase classname="ictInternshipPortal.LoginTC" name="TC_IP_1_2_2" time="1.984"/>
  <testcase classname="ictInternshipPortal.LoginTC" name="TC_IP_1_2_3" time="1.381"/>
  <testcase classname="ictInternshipPortal.LoginTC" name="TC_IP_1_2_6" time="1.784"/>
  <testcase classname="ictInternshipPortal.LoginTC" name="TC_IP_1_2_4" time="20.603"/>
  <testcase classname="ictInternshipPortal.LoginTC" name="TC_IP_1_2_5" time="40.969">
    <failure message="expected [Please enter your password] but found [Password must be at least 8 characters]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Please enter your password] but found [Password must be at least 8 characters]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at ictInternshipPortal.LoginTC.TC_IP_1_2_5(LoginTC.java:95)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- TC_IP_1_2_5 -->
</testsuite> <!-- ictInternshipPortal.LoginTC -->
